lib/externalGraderLocal.js

```diff
@@ -89,6 +89,8 @@ class Grader {
             (callback) => {
                 docker.createContainer({
                     Image: question.external_grading_image,
+                    // Convert {key: 'value'} to ['key=value'] and {key: null} to ['key'] for Docker API
+                    Env: Object.entries(question.external_grading_environment).map(([k, v]) => v === null ? k : `${k}=${v}`),
```

Comment:
@mwest1066 remind me what the latest with PG is - are we still running off of https://github.com/PrairieLearn/PrairieGrader in production, or are we using the local `grader_host` directory?

@tdy pending the above, you may have to add code in PrairieGrader to handle this, but at a minimum, you'll need to update https://github.com/PrairieLearn/PrairieLearn/blob/master/lib/externalGraderSqs.js and https://github.com/PrairieLearn/PrairieLearn/blob/master/grader_host/index.js to handle this as well.

Reply:
> at a minimum, you'll need to update `lib/externalGraderSqs.js` and `grader_host/index.js`

done in 2a2061c28db7be6194e913ce34950f3747bf4389 (I think)